projects/step-frontend/src/lib/modules/timeseries/components/dashboard/dashboard.component.html

```diff
@@ -1,7 +1,18 @@
 @if (dashboard) {
   <div class="page-container">
     <div class="header-container">
-      <div></div>
+      <div>
+        <button class="menu-btn" #menuTrigger="matMenuTrigger" [matMenuTriggerFor]="menu" mat-raised-button>
+          <step-icon name="settings"></step-icon>
+          Settings
+        </button>
+        <mat-menu #menu="matMenu">
+          <step-ts-resolution-picker
```

Comment:
Please replace this with our step-time-input component

Reply:
@neogucky I did check the component. I've seen the method `autoDetermineDisplayMeasure()` but I don't see clearly how can I have the input recognize and fill the appropriate value. For example if I put in the model 3600, I should see 1 hour - this is not the case. Do you know how I can have the autodetect triggered?

@dvladir it seems you did the component, maybe it's easier for you to help

Reply:
@lucianbaciu `autoDetermineDisplayMeasure()` is invoked, when model is assigned to component, and no `displayMeasure` is specified

Reply:
I did replace it